aws/provider_test.go

```diff
@@ -498,6 +498,14 @@ func testAccGetThirdRegionPartition() string {
 	return "aws"
 }
 
+func testAccRDSCACertificateID() string {
+	v := os.Getenv("AWS_RDS_CA_CERTIFICATE_IDENTIFIER")
+	if v == "" {
+		return "rds-ca-2019"
+	}
+	return v
+}
+
```

Comment:
Longterm we can consider https://github.com/terraform-providers/terraform-provider-aws/issues/15221 -- maybe I'll just create that right now. Otherwise, we will want to at least ensure this is documented in the Maintaining Guide: https://github.com/terraform-providers/terraform-provider-aws/blob/master/docs/MAINTAINING.md#environment-variable-dictionary

Reply:
Submitted this for now or later: https://github.com/terraform-providers/terraform-provider-aws/pull/15789

Reply:
Done!